Clarify non-normative text. (#3197)
modelica/ModelicaSpecification: chapters/connectors.tex

  \item
    If the variable on the other side of the \lstinline!connect!-equation is \lstinline!input! or \lstinline!output! the new component will be either \lstinline!input! or \lstinline!output! to satisfy the restrictions in \cref{restrictions-of-connections-and-connectors} for a non-expandable connector.
    \begin{nonnormative}
    The general rule ensures consistency for inside and outside connectors, and handles multiple connections to the new component.

>     For the simple case of no other connections involving these variables it can be explained as follows:

Reviewer: Inside a paragraph (unlike before a listing or an item list) I find it odd to have a colon between sentences.
```suggestion
    For the simple case of no other connections involving these variables it can be explained as follows.
```

>     For the simple case of no other connections involving these variables it can be explained as follows.
>     If the existing side refers to an inside connector (i.e.\ a connector of a component) the new variable will copy its causality, i.e.\ \lstinline!input! if \lstinline!input! and \lstinline!output! if \lstinline!output!, since the expandable connector must be an outside connector.

Reviewer: Maybe it would make sense to combine these two sentences into one?
```suggestion
    In the simple case of no other connections involving these variables and the existing side referring to an inside connector (i.e., a connector of a component), the new variable will copy its causality (i.e., \lstinline!input! if \lstinline!input! and \lstinline!output! if \lstinline!output!) since the expandable connector must be an outside connector.
```

Author: Could do.
